{{cookiecutter.project_slug}}/.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,22 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: 0 10 * * 1
```

Comment:
can you comment this with like `# min, hour, day(month), week, day(week)` or something? Also `0 10 * * MON` is equivalent and maybe more readable